instrumentation/elasticsearch/lib/opentelemetry/instrumentation/elasticsearch/patches/client.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'uri'
+
+module OpenTelemetry
+  module Instrumentation
+    module Elasticsearch
+      module Patches
+        # Module to prepend to Elasticsearch::Client for instrumentation
+        module Client
+          def self.included(base) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength
+            base.class_eval do # rubocop:disable Metrics/BlockLength
+              alias_method :perform_request_without_otel, :perform_request
```

Comment:
I know this is still in draft, but wanted to provide the heads up that we have been consistently sticking to prepending instead of method aliasing for our instrumentation patches.

Reply:
Updated the PR and moved away from `class_eval` 👍 

Thanks @robertlaurin!